src/main/java/com/fakng/fakngagrgtr/parser/stripe/StripeParser.java

```diff
@@ -0,0 +1,132 @@
+package com.fakng.fakngagrgtr.parser.stripe;
+
+import com.fakng.fakngagrgtr.parser.HtmlParser;
+import com.fakng.fakngagrgtr.parser.LocationProcessor;
+import com.fakng.fakngagrgtr.persistent.company.CompanyRepository;
+import com.fakng.fakngagrgtr.persistent.vacancy.Vacancy;
+import com.gargoylesoftware.htmlunit.WebClient;
+import com.gargoylesoftware.htmlunit.html.*;
+import lombok.SneakyThrows;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class StripeParser extends HtmlParser {
+
+    private final static String VACANCY_URL = "https://stripe.com/jobs/listing";
+    private final static String JOBS_PAGINATION_LIST_XPATH = "/html/body/div/section[4]/div/div[2]/div/div/nav/ul/li[7]/a";
+    private final static String VACANCY_TITLE_XPATH = "/html/body/section[1]/div/div[2]/div/div/section/header/h1";
+
+    public StripeParser(WebClient htmlWebClient,
+                        CompanyRepository companyRepository,
+                        LocationProcessor locationProcessor,
+                        @Value("${url.stripe}") String url) {
+        super(htmlWebClient, companyRepository, locationProcessor);
+        this.url = url;
+    }
+
+    @PostConstruct
+    public void init() {
+        initBase();
+    }
+
+    @Override
+    protected String getCompanyName() {
+        return "Stripe";
+    }
+    @Override
+    public List<Vacancy> getAllVacancies() throws IOException {
+        HtmlPage page = downloadPage(String.format(url, 0));
+        int pageCount = getPageCount(page);
+        List<Vacancy> vacancyList = new ArrayList<>();
+
+        for (int i = 0; i <= pageCount; i++) {
+            page = downloadPage(String.format(url, pageCount));
+            vacancyList.addAll(fillVacancy2(getLinksFromPage(page)));
+        }
+
+        return vacancyList;
+    }
+
+    private int getPageCount(HtmlPage page) {
+        return Integer.parseInt(page.getByXPath(JOBS_PAGINATION_LIST_XPATH).get(0).toString().replaceAll("\\D+", "")) / 100;
```

Comment:
This line can be better

First of all, you can get DomNode text with some method call, without using toString(). It might save you from doing those replacements potentially

Second, what is 100? For me, it's just a magic number. Of course, it's kinda page size, but we don't put any numbers just like this. It should be moved to a constant with a descriptive name

Third, I just opened Stripe careers. There were 5 pages available on UI. This link you refer to with this XPath returns 600. You divide it by 100 and get 6. Is it the correct number?